lib/babel-options-util.js

```diff
@@ -0,0 +1,476 @@
+const VersionChecker = require("ember-cli-version-checker");
+const clone = require("clone");
+const semver = require("semver");
+
+const APP_BABEL_RUNTIME_VERSION = new WeakMap();
+
+const findApp = require("./find-app");
+const defaultShouldIncludeHelpers = require("./default-should-include-helpers");
+
+module.exports = {
+  _getPresetEnv(config, project) {
```

Comment:
Let's roll this back to an earlier format. We do need to export, but I'd like to be able to call the functions _without_ `this`.

Something like:

```js
function getPresetEnv(config, project) {
}

//...snip...

module.exports = {
  _getPresetEnv: getPresetEnv,
}
```

Reply:
sure thing, if we do that then we will have to export all the helper methods, which I am not sure if its helpful. May be I could just export the functions that I am leveraging in the tests rather than all of them